internal/cmd/cloud.go

```diff
@@ -25,6 +25,14 @@ import (
 	"github.com/spf13/pflag"
 )
 
+// ErrUserUnauthenticated represents an authentication error when trying to use
```

Comment:
Do we really need to make this error public? 🤔  I'm wondering because I see it's used by no other package, and it's `internal` so other projects like extensions could not use it.

Reply:
No, we don't. I've changed it, good catch!